dist/index.js

```diff
@@ -113,7 +122,7 @@ function terraform(params) {
   }
 
   core.startGroup('Run terraform plan');
-  const tfp = terraform('plan -out=terraform.plan');
+  const tfp = terraform('plan -out=terraform.plan -lock=' + terraformLock);
```

Comment:
Is the conversion `boolean → string` implicit?

Reply:
It never is a boolean in the first place, just looks like one :wink: